### ISSUE TITLE
Join Table Between Outputs and Claims

### ISSUE THREAD

**Issue by tiger5226** [lbryio/chainquery#8]
A claim has at least one output, the original one that created the claim. However, it can also have multiple updates. Creating a join table between these will allow for better access to important information from previous updates or the original claim.

**Comment by tiger5226** [comment]
OutputClaim table has been added and migration created 0b8ae9337508b53e91ef3ce2bc293ee45cdd6a96

Still needs processing logic to make sure these are created.

**Comment by tiger5226** [comment]
A join table is not needed and is unnecessary overhead so I removed that. Instead I have added claimid to the output table so that any output can reference a claim. This will allow left join queries to obtain all the outputs for a claimid. This will be needed for solving https://github.com/lbryio/lighthouse/issues/63. To solve that issue we need to be able to take an outpoint and get the correct claimid. Currently it is calculating the claimid based off of the outpoint. However, the outpoint could be an update to a claim causing the wrong claimid to be calculated as the claimid is from the original outpoint. With this change we can query the outpoint and grab the claimid.
